src/Microsoft.Diagnostics.EventFlow.Inputs.PerformanceCounter/Configuration/PerformanceCounterConfiguration.cs

```diff
@@ -3,42 +3,87 @@
 //  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
+using System;
+
 namespace Microsoft.Diagnostics.EventFlow.Configuration
 {
     // !!ACTION!!
     // If you make any changes here, please update the README.md file to reflect the new configuration
     public class PerformanceCounterConfiguration
     {
-        public string CounterCategory { get; set; }
+        private static readonly string DotNetProcessIdCounterCategory = ".NET CLR Memory";
```

Comment:
Should add a comment explaining these counters are special that you need to special-map them.